src/components/enterprise-page/data/hooks.js

```diff
@@ -3,32 +3,48 @@ import { useParams } from 'react-router-dom';
 import { logError } from '@edx/frontend-platform/logging';
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 
-import { fetchEntepriseCustomerConfig } from './service';
+import { fetchEnterpriseCustomerConfig } from './service';
 
-const defaultBorderColor = '#007D88';
-const defaultBackgroundColor = '#D7E3FC';
+export const defaultBorderColor = '#007D88';
+export const defaultBackgroundColor = '#D7E3FC';
 
+const defaultBrandingConfig = {
+  logo: null,
+  bannerBackgroundColor: defaultBackgroundColor,
+  bannerBorderColor: defaultBorderColor,
+};
+
+/**
+ * If enterpriseSlug is passed, it will be used instead of the useParams hook output
+ * @param {string} [enterpriseSlug] enterprise slug.
+ * @returns {object} EnterpriseConfig
+ */
 // eslint-disable-next-line import/prefer-default-export
-export function useEnterpriseCustomerConfig() {
-  const { enterpriseSlug } = useParams();
+export function useEnterpriseCustomerConfig(enterpriseSlug) {
+  const slugFromUseParams = () => {
```

Comment:
I think it's OK for us to always pass in `enterpriseSlug`. If no `enterpriseSlug` is provided, we should be able to return `null`.

That way, we can always have consumers of this hook utilize `useParams` to get the `enterpriseSlug` to pass in. One less dependency for the `useEnterpriseCustomerConfig` hook:

For example, in `<EnterprisePage />`:

```
const { enterpriseSlug } = useParams();
const enterpriseConfig = useEnterpriseCustomerConfig(enterpriseSlug);
```

Reply:
got it, this is even better, on it!